ag/ag.proto

```diff
@@ -199,6 +199,25 @@ message Issue {
     uint64 issueNumber  = 4;  // Issue number on scm. Needed for associating db issue with scm issue
 }
 
+message PullRequest {
+    enum Stage {
+        NONE    = 0;
+        DRAFT   = 1;
+        REVIEW  = 2;
+        APPROVED = 3;
+    }
+    uint64 ID                   = 1;
+    // TODO(meling): When a naming convention has been decided, change this field's name.
+    uint64 externalRepositoryID = 2; // Represents the external repository ID
```

Comment:
We should update this to `scmRepositoryID` pending a final decision, but so far `scm` is my preference.